libraries/chain/apply_context.cpp

```diff
@@ -17,6 +17,10 @@ using boost::container::flat_set;
 namespace eosio { namespace chain {
 
 static inline void print_debug(account_name receiver, const action_trace& ar) {
+   if (eosio::chain::chain_config::deep_mind_console_enabled) {
+      return;
```

Comment:
Remove. This is unnecessary because if `contracts-console` is disabled, then `ar.console.empty()` will always be true and this function will do nothing.